app/models/proposal.rb

```diff
@@ -31,8 +31,12 @@ class Proposal < ActiveRecord::Base
   acts_as_taggable
 
   has_many :steps
-  has_many :individual_steps, ->{ individual }, class_name: 'Steps::Individual'
-  has_many :approvers, through: :individual_steps, source: :user
+  has_many :individual_steps, ->{ individual }, class_name: "Steps::Individual"
+  has_many :approval_steps, class_name: "Steps::Approval"
+  has_many :purchase_steps, class_name: "Steps::Purchase"
+  has_many :approvers_and_purchasers, through: :individual_steps, source: :user
```

Comment:
renamed at 493914b0a56c41e319cd43ace5b31ec223ea9169 -- let me know what you think!